src/sst/elements/ember/Makefile.am

```diff
@@ -12,6 +12,8 @@ compdir = $(pkglibdir)
 comp_LTLIBRARIES = libember.la
 
 libember_la_SOURCES = \
+	example/motifs/emberexample.h
```

Comment:
need the "\\" after these

Reply:
fixed with changeset [f5a4ac1](https://github.com/sstsimulator/sst-elements/pull/1948/commits/f5a4ac14692fcad873abe6a944766bceee3d4565)